packages/aws-cdk/lib/cli/user-input.ts

```diff
@@ -727,6 +727,13 @@ export interface DeployOptions {
    */
   readonly hotswap?: boolean;
 
+  /**
+   * Number of seconds to wait for a hotswap operation to complete. Normally this applies to resource stabilization.
+   *
+   * @default - undefined
```

Comment:
"undefined" is not good value for `@default`. *I already knew that*. Tell me what it does instead.

Reply:
This is code generated from the yargs definitions, so every string option gets `undefined` as its `@default` value. I don't believe there's a way to inject here a "description of the default".

Reply:
Not yet, we could add that though. We could also just explicitly set a default global timeout as part of the config.